clients/client-python/requirements.txt

```diff
@@ -3,4 +3,8 @@
 
 # the tools to publish the python client to Pypi
 requests
-dataclasses-json
\ No newline at end of file
+dataclasses-json
+readerwriterlock==1.0.9
+fsspec==2024.3.1
+pyarrow
```

Comment:
Do we need to specify the version of `pyarrow` here?

Reply:
The pyarrow version is already specified in `requirement-dev.txt`, which I think is no longer needed here.